activate.sh

```diff
@@ -262,7 +273,7 @@ autoenv_source() {
 # @description Function to override the 'cd' builtin
 autoenv_cd() {
 	local _pwd=${PWD}
-	if \command -v chdir >/dev/null 2>&1 && \chdir "${@}" || \builtin cd "${@}"; then
+	if __autoenv_cd "${@}"; then
```

Comment:
All args are being passed here to the new wrapper function. Taking a shot in the dark at how this change resulted in #254, I'm guessing this is where the zero args call to `cd`  back home goes wrong. No change should be required here, but rather in `__autoenv_cd` which needs to support forwarding zero args to the built-in.